content/sql/concepts/window-functions/terms/lag/lag.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'LAG()'
+Description: 'Accesses previous rows data without the need for self join.'
+Subjects: 
+  - 'Data Science'
+Tags: 
+  - 'SQL Server'
+CatalogContent: 
+  - 'data-science-foundations'
+  - 'catalog/subject/data-science'
+---
+
+The **LAG()** window function facilitates access to previous rows based on the offset argument. It can be particularly useful when a comparison of a previous value is necessary without the use of a self join. There is a similarity to the `LEAD()` function with the difference being the accessible rows. `LEAD()` accesses subsequent rows while `LAG()` accesses previous rows.
+
+## Syntax
+```pseudo
+LAG (expression [, offset] [, default])  
+OVER ( [ partition_by ] order_by )
+```
+expression - 
```

Comment:
The following line appear incomplete, missing descriptions for the indicated elements in the syntax block.